src/client/app/containers/DashboardContainer.ts

```diff
@@ -19,6 +19,10 @@ import { TimeInterval } from '../../../common/TimeInterval';
 
 
 function mapStateToProps(state: State) {
+	// TODO this is a hack to get the unit state set up correctly.
+	// It is not really safe to set to an arbitrary value but
+	// the Unit route is not done yet.
+	state.graph.selectedUnit = 1;
```

Comment:
The unit route was created in PR #739. Do you want to change the `selectedUnit` here? I think the current code is ok since we still need to replace this when the unit menu is completed. But changing now will make it safer.

Reply:
I'm not certain I understand but let me take a shot. The units route that exists gets the units. If what you mean is that I could find a desired unit from that information rather than using an arbitrary number then I agree. I didn't do that because you have to manually match the unit to the meter(s) being graphed so I left it as a number. And, yes, this will go away soon when the unit is selected by the user. Let me know if you think this is resolved or I should consider something else.

Reply:
Yes, it makes sense to me. I think we can leave it as it is now. Thank you for your explanation.